examples/anim/lv_example_anim_3.c

```diff
@@ -0,0 +1,152 @@
+#include "../lv_examples.h"
+#if LV_BUILD_EXAMPLES && LV_USE_SLIDER && LV_USE_CHART && LV_USE_BTN
+
+/**
+ * the example show the use of cubic-bezier3 in animation.
+ * the control point P1,P2 of cubic-bezier3 can be adjusted by slider.
+ * and the chart shows the cubic-bezier3 in real time. you can click
+ * run button see animation in current point of cubic-bezier3.
+ */
+
+#define CHART_POINTS_NUM 256
+
+struct {
+    lv_obj_t * anim_obj;
+    lv_obj_t * chart;
+    lv_chart_series_t * ser1;
+    lv_obj_t * p1_slider;
+    lv_obj_t * p1_label;
+    lv_obj_t * p2_slider;
+    lv_obj_t * p2_label;
+    lv_obj_t * run_btn;
+    uint16_t p1;
+    uint16_t p2;
+    lv_anim_t a;
+} ginfo;
+
+static int32_t anim_path_bezier3_cb(const lv_anim_t * a);
+static void refer_chart_cubic_bezier(void);
+static void run_btn_event_handler(lv_event_t * e);
+static void slider_event_cb(lv_event_t * e);
+static void page_obj_init(lv_obj_t * par);
+static void anim_x_cb(void * var, int32_t v);
+
+/**
+ * create a animation
+ */
+void lv_example_anim_3(void)
+{
+    page_obj_init(lv_scr_act());
+
+    lv_anim_init(&ginfo.a);
+    lv_anim_set_var(&ginfo.a, ginfo.anim_obj);
+    lv_anim_set_values(&ginfo.a, lv_obj_get_x(ginfo.anim_obj), lv_obj_get_x(ginfo.anim_obj) + 400);
```

Comment:
I'd rather animate between `10` and `lv_obj_get_width(parent) - lv_obj_get_width(ginfo.anim_obj) - 10` to make work well on every screen sizes.